src/Radicle/Internal/Pretty.hs

```diff
@@ -23,6 +26,9 @@ instance Pretty Reference where
 instance Pretty Ident where
     pretty (Ident i) = pretty i
 
+instance Pretty Type where
+  pretty = fromString . drop 1 . show
```

Comment:
Should this use `typeToValue`?

Reply:
Yes this and `typeToValue` will both just use https://github.com/oscoin/radicle/pull/242/files#diff-dd5f436b10a11b14706cef020de15c04R44 when that is merged and I rebase this PR.